symphony/bdk/core/activity/command.py

```diff
@@ -72,6 +72,16 @@ def __init__(self, name, requires_mention_bot, callback):
         self._name = name
         self._requires_mention_bot = requires_mention_bot
         self._callback = callback
+        self._description = description
+
+    @property
+    def name(self) -> str:
+        return self._name
+
+    def build_command_description(self) -> str:
+        if self._requires_mention_bot:
```

Comment:
This could even be a one-liner: 
`return self._description + " (mention required)" if self._requires_mention_bot else self._description + " (mention not required)"`
But this way is fine, too.